api/api.yaml

```diff
@@ -1619,6 +1615,44 @@ paths:
                       items:
                         type: string
                       description: The scopes to use for the connection (optional).
+                oauth2AuthorizationCode:
+                  type: object
+                  properties:
+                    accessToken:
+                      type: object
+                      properties:
+                        token:
+                          type: string
+                          description: The access token to use for the connection.
+                        expiresAt:
+                          type: string
+                          format: date-time
+                          description: The expiration date of the access token.
+                        issuedAt:
+                          type: string
+                          format: date-time
+                          description: The date the access token was issued.
+                      description: access token for the connection.
+                    refreshToken:
+                      type: object
+                      properties:
+                        token:
+                          type: string
+                          description: The refresh token to use for the connection.
+                        issuedAt:
+                          type: string
+                          format: date-time
+                          description: The date the refresh token was issued.
+                        expiresAt:
+                          type: string
+                          format: date-time
+                          description: The expiration date of the refresh token.
+                      description: The refresh token to use for the connection.
+                    scopes:
+                      type: array
+                      items:
+                        type: string
+                      description: The scopes to use for the connection (optional).
```

Comment:
```suggestion
                      description: The scopes for the tokens.
```

Reply:
No need to add `(optional)` to the description, since that should be indicated by the presence or absence of `required` list in the openapi spec itself.